repl.html

```diff
@@ -31,18 +31,19 @@
 </div>
 <div>
 <!-- TODO(max): Add button to save to/load from disk. -->
-<button id="clear-local-storage">Clear LocalStorage</button>
+<button id="clear-local-storage">Clear</button>
 </div>
 <div id="output" style="height: 400px; overflow: auto;">
 Output:
 </div>
 <div>
-<code>>>> </code><input id="input" type="text" />
+<code><span id="prompt-string">>>></span></code><input id="input" type="text" />
 </div>
 <script type="module">
 "use strict";
 
 function updateHistory(inp, out) {
+    const wrap = document.createElement("div");
```

Comment:
Do you use wrap? If not, we can remove

Reply:
Wrap is used to avoid having the flex gap apply separately to result and not(.result) elements in #output. Grouping every pair of input-output makes it easy to use flex gap rather than margin/padding